### ISSUE TITLE
How do I specify a web page as part of liveshow?

### ISSUE THREAD

**Issue by jamesagada** [KenT2/pipresents-gapless#148]
I want to add a web page to the liveshow and there is no clear guidance. Do I just drop an html file? 
I need a specific page to be opened.

**Comment by KenT2** [comment]
Yes just include a .htm or .html file in the live tracks directory.
